### PR TITLE
Pagination and grammar fixes for 500.html file

### PR DESCRIPTION
**Pull request by Shaikha1212**
resolves https://github.com/CompanionshipCare/companionship-care/issues/15

### CODE REVIEW

**Comment by Shaikha1212** [comment]
Just wanted to mention that when running the manage.py test line in the terminal all tests pass

File: project/circles/views.py [hunk 1]
```diff
@@ -122,6 +126,18 @@ def get_context_data(self, **kwargs):
 
         context["add_activity_form"] = ActivityModelForm
 
+        try:
+            activities_page = paginator.page(page)
+            print("got page")
+            print(activities_page)
```
Comment (brylie) [inline project/circles/views.py:132]: Let's remove these print statements.

**Review by brylie — CHANGES_REQUESTED** [review]

File: project/circles/templates/circles/circle_detail.html [hunk 2]
```diff
@@ -103,6 +103,42 @@ <h2>
...
+    <!--Pagination-->
+    <nav aria-label="Page navigation example">
+        <ul class="pagination justify-content-center">
+          {% if activity_page.has_previous %}
+            <li class="page-item">
+              <a class="page-link" href="?page={{ activity_page.previous_page_number }}">Previous</a>
+            </li>
+          {% else %}
+            <li class="page-item disabled">
+              <a class="page-link" href="#" tabindex="-1" aria-disabled="True">Previous</a>
```
Comment (brylie) [inline project/circles/templates/circles/circle_detail.html:115]: Remember the i18n tag here.
```suggestion
              <a class="page-link" href="#" tabindex="-1" aria-disabled="True">
                  {% translate "Previous" %}
              </a>
```

File: project/circles/templates/circles/circle_detail.html [hunk 3]
```diff
@@ -103,6 +103,42 @@ <h2>
...
+                  {{ i }}
+                </span>
+              </li>
+            {% else %}
+              <li class="page-item"><a class="page-link" href="?page={{ i }}">{{ i }}</a></li>
+            {% endif %}
+          {% endfor %}
+          {% if activity_page.has_next %}
+            <li class="page-item">
+              <a class="page-link" href="?page={{ activity_page.next_page_number }}">Next</a>
```
Comment (brylie) [inline project/circles/templates/circles/circle_detail.html:131]: Add i18n:
```suggestion
              <a class="page-link" href="?page={{ activity_page.next_page_number }}">
                  {% translate "Next" %}
              </a>
```

File: project/circles/templates/circles/circle_detail.html [hunk 4]
```diff
@@ -103,6 +103,42 @@ <h2>
...
+            </li>
+          {% else %}
+            <li class="page-item disabled">
+              <a class="page-link" href="#" tabindex="-1" aria-disabled="True">Next</a>
```
Comment (brylie) [inline project/circles/templates/circles/circle_detail.html:135]: i18n:
```suggestion
              <a class="page-link" href="#" tabindex="-1" aria-disabled="True">
                  {% translate "Next" %}
              </a>
```

**Review by brylie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**